File: Card/src/main/java/com/braintreepayments/api/CardTokenizeCallback.java
package com.braintreepayments.api;

/**
 * Callback for receiving result of {@link CardClient#tokenize(Card, CardTokenizeCallback)}.
 */
public interface CardTokenizeCallback {

    /**
     * @param cardResult a {@link CardResult} containing a {@link CardNonce} or {@link Exception}
     */
    void onCardResult(CardResult cardResult);
}

File: Card/src/main/java/com/braintreepayments/api/CardTokenizeCallback.kt
package com.braintreepayments.api

/**
 * Callback for receiving result of [CardClient.tokenize].
 */
fun interface CardTokenizeCallback {

    /**
     * @param cardResult a [CardResult] containing a [CardNonce] or [Exception]
     */
    fun onCardResult(cardResult: CardResult?)
}
